### PR TITLE
Feature/power bi

### PR DESCRIPTION
**Pull request by Lance-Simeon**
Need to update branch pointers before merging

### CODE REVIEW

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: doesn't Publish-PowerBISimeonSyncRun require this?

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: this doesn't seem right...where is Reporting.yml? I'd expect this to call SimeonCloudReportingBackfill

Comment (Lance-Simeon) [reply BackfillPowerBI.yml]: yeah, you were right, had missed a push.

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: This will be enough for an org with 100 tenants?

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: Just use all these env vars inline in the script instead of passing as env vars

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: Just use inline in the script - the reason being for consistency - you already use System.CollectionUri in-line in the script

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: This will fail the job and the next two write-error lines won’t execute. I’d think you’d want this after the other two write-errors (in this case we actually do want it to fail the job)

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: Invoke-SimeonSyncPowerBIBackfill for consistency

File: BackfillPowerBI.yml
Comment (Lance-Simeon) [reply BackfillPowerBI.yml]: Don't I need to keep this one since its secret?

File: BackfillPowerBI.yml
Comment (Lance-Simeon) [reply BackfillPowerBI.yml]: What do you suggest it be?

File: BackfillPowerBI.yml
Comment (jeff-simeon) [reply BackfillPowerBI.yml]: yes

File: BackfillPowerBI.yml
Comment (jeff-simeon) [reply BackfillPowerBI.yml]: I don't know how long is runs today....I defer to you to do the math on how long a backfill for 100 tenants would take...then multiply that by 10 I suppose

**Review by jeff-simeon — DISMISSED** [review]

File: BackfillPowerBI.yml
Comment (Lance-Simeon) [reply BackfillPowerBI.yml]: Takes 2.5 minutes for 8 tenants to backfill 72 hours. Lets say 30 minutes for 100 tenants, maybe 3 hours?

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml:42]: how are these env variables getting populated? I would have expected them to all just look like $(PowerBIXyz)

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: remove this env var

**Review by jeff-simeon — DISMISSED** [review]

File: BackfillPowerBI.yml
Comment (jeff-simeon) [inline BackfillPowerBI.yml:27]: Would call MaxParallelism

**Review by jeff-simeon — DISMISSED** [review]

**Review by jeff-simeon — DISMISSED** [review]
Make sure to change branch before merge

**Review by jeff-simeon — APPROVED** [review]

**Review by david-simeon — APPROVED** [review]

**Pull request merged**
